src/main/app/e2e/step-definitions/common.ts

```diff
@@ -3,7 +3,13 @@
  * SPDX-License-Identifier: EUPL-1.2+
  */
 import { Given, When, Then } from "@cucumber/cucumber";
-import { CustomWorld } from "support/worlds/world";
+import { CustomWorld } from "../support/worlds/world";
+import { waitForHealthCheck } from "../support/utils/health";
+
+
+Given("Zac is live", { timeout: 5 * (60 * 1000) }, async function (this: CustomWorld) {
+    await waitForHealthCheck(this)
+});
 
 Given("{string} navigates to {string}", { timeout: 60 * 1000 }, async function (this: CustomWorld, user, url) {
```

Comment:
So here, I'd like it if we can load the "url" from an environment property using the string.